crates/ruma-events/src/enums.rs

```diff
@@ -8,6 +8,20 @@ use serde_json::value::RawValue as RawJsonValue;
 
 use super::room::encrypted;
 
+/// [Stripped state] event types which the spec recommends that servers should send so that clients
+/// can identify rooms
+///
+/// [Stripped state]: https://spec.matrix.org/latest/client-server-api/#stripped-state
```

Comment:
I believe that this is clearer, if you agree:
```suggestion
/// Event types that servers should send as [stripped state] to help clients identify a room when they can't access the full room state.
///
/// [stripped state]: https://spec.matrix.org/latest/client-server-api/#stripped-state
```